packages/devtools_app/lib/src/service_extensions.dart

```diff
@@ -258,7 +339,14 @@ final toggleSelectWidgetMode = ToggleableServiceExtensionDescription<bool>._(
 final enableOnDeviceInspector = ToggleableServiceExtensionDescription<bool>._(
   extension: 'ext.flutter.inspector.enable',
   description: 'Enable on-device inspector',
-  icon: createImageIcon('icons/general/locate@2x.png'),
+  disabledIcon: const ThemedImageIcon(
```

Comment:
example of a case where ThemedImageIcon shouldn't be used as the light and dark mode assets are the same. Create a separate simpler ImageIcon class or just use createImageIcon as before. Specifying the same asset twice for this case just makes the code harder to read.